SPECS/kata-containers-cc/kata-containers-cc.spec

```diff
@@ -0,0 +1,281 @@
+%global runtime_make_vars       DEFSERVICEOFFLOAD=true \\\
+                                SKIP_GO_VERSION_CHECK=1
+
+%global agent_make_vars         LIBC=gnu
+
+%global debug_package %{nil}
+
+Name:         kata-containers-cc
+Version:      0.4.0
+Release:      1%{?dist}
+Summary:      Kata Confidential Containers
+License:      ASL 2.0
+Vendor:       Microsoft Corporation
+URL:          https://github.com/microsoft/kata-containers
+Source0:      https://github.com/microsoft/kata-containers/archive/refs/tags/cc-%{version}.tar.gz#/%{name}-%{version}.tar.gz
+Source1:      https://github.com/microsoft/kata-containers/archive/refs/tags/%{name}-%{version}.tar.gz
+
+Source2:      %{name}-%{version}-cargo.tar.gz
+Source3:      containerd-for-cc-override.conf
+Source4:      mariner-coco-build-uvm-image.sh
+Source5:      mariner-coco-build-uvm-rootfs.sh
+
+ExclusiveArch: x86_64
+
+BuildRequires:  golang
+BuildRequires:  make
+BuildRequires:  gcc
+BuildRequires:  protobuf-compiler
+BuildRequires:  dracut
+BuildRequires:  cargo
+BuildRequires:  rust
+BuildRequires:  git
+BuildRequires:  sudo
+BuildRequires:  openssl-devel
+BuildRequires:  iptables-devel
+BuildRequires:  wget
+BuildRequires:  perl-FindBin
+BuildRequires:  perl-lib
+BuildRequires:  libseccomp-devel
+BuildRequires:  kernel-uvm-devel
+BuildRequires:  parted
+BuildRequires:  opa >= 0.50.2
+
+Requires:  systemd
+Requires:  iptables
+Requires:  parted
```

Comment:
If we had a subpackage for the UVM build package we may get rid of some Requires for kata-containers-cc

Reply:
Resp., I'm seeing we have a tools subpackage now. Are we sure we need parted and qemu-img for kata-containers-cc? I think those may be moved to the tools pcakage? Also, kernel-uvm

Reply:
Requirements moved to their respective subpackages